.github/workflows/publish_connectors.yml

```diff
@@ -106,7 +93,25 @@ jobs:
             {
               "channel": "#connector-publish-failures",
               "username": "Connectors CI/CD Bot",
-              "text": "🚨 Publish workflow failed:\n ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+              "text": "🚨 Publish workflow failed:\n ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} \n merged by ${{ github.actor }} (<@${{ steps.match-github-to-slack-user.outputs.slack_user_ids }}>). "
             }
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.PUBLISH_ON_MERGE_SLACK_WEBHOOK }}
+
+  notify-failure-pager-duty:
+    name: "Notify PagerDuty on Publish Failures"
+    runs-on: ubuntu-latest
+    needs:
+      - publish_connectors
+    if: ${{ always() && contains(needs.*.result, 'failure') && github.ref == 'refs/heads/master' }}
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v4
+      - name: Notify PagerDuty
+        id: pager-duty
+        uses: ./.github/actions/send-pager-duty-event
+        with:
+          integration_key: ${{ secrets.PAGER_DUTY_PUBLISH_FAILURES_INTEGRATION_KEY }}
```

Comment:
Can you put a comment in here saying which team's integration key this is? (I assume it's tooling's?)

Reply:
Integration keys are not per team, they are per serivce.
We have a [Connector Publish Pipeline service](https://airbyte.pagerduty.com/service-directory/P5GNI5T).
I set up an "Events API V2" integration (basically a webhook) which our GH workflow POST to.
This creates an alert. Then how this alert is managed is handled in PD settings with escalation policies etc.

Reply:
Okay cool thanks for clarifying. I think it would be nice if the secret name had the service name in it, or if there was a comment saying exactly which service it's attached to. (I can basically tell by the name but would be nice not to have to guess!)

Reply:
@clnoll I'll add a comment with an URL to the integration.

Reply:
https://github.com/airbytehq/airbyte/pull/43846